lib/Service/ApiService.php

```diff
@@ -127,6 +124,8 @@ public function create($fileId = null, $filePath = null, ?string $token = null,
 			return new DataResponse('Failed to create the document session', 500);
 		}
 
+		/** @var Document $document */
+
```

Comment:
Nitpicking, but this empty line is superfluous, no?

Reply:
This is not the var of the left line but an indicator for psalm not detecting that document is never undefined in this case (which seems like a false positive).